tox.ini

```diff
@@ -9,7 +9,9 @@ changedir = {toxinidir}/raiden
 deps =
     -rrequirements.txt
     pdbpp
+    pep8
```

Comment:
Go with flake8 then, it's quite a complete tool and will help us a lot in keeping our styling checked. Moreover we need consistency, can't have each dev trying out a different tool.